src/components/DynamicDimension/DynamicDimension.js

```diff
@@ -1,29 +1,48 @@
 import React, { useEffect, useState } from 'react'
 import PropTypes from 'prop-types'
 
+import { useDataQuery } from '@dhis2/app-runtime'
 import ItemSelector from './ItemSelector'
-import { apiFetchItemsByDimension } from '../../api/dimensions'
+
+const itemsByDimensionQuery = {
+    itemsByDimension: {
+        resource: 'dimensions',
+        id: ({ id }) => `${id}/items`, // TODO replace with the new way when available
+        params: {
+            fields: 'id,displayName~rename(name)',
+            order: 'displayName:asc',
+            //paging: false, // XXX needed?!
```

Comment:
needs addressing?

Reply:
Could be fixed as part of https://jira.dhis2.org/browse/DHIS2-9274